src/Surfnet/ServiceProviderDashboard/Domain/Entity/ManageEntity.php

```diff
@@ -180,6 +180,11 @@ public function getAllowedIdentityProviders()
         return $this->allowedIdentityProviders;
     }
 
+    public function clearAllowedIdentityProviders()
+    {
+        $this->allowedIdentityProviders = new AllowedIdentityProviders([], true);
+    }
```

Comment:
Gets the job done for certain. Have you also considered adding a `clear` method to the `AllowedIdentityProviders` VO? That could unset all data and allows the VO itself to determine what a cleared state really means. That way you do not actually need to add this method, but use the getter and call `clear` on the VO.

Reply:
i had not, but i like the idea :-)

It has been done 🧙